### DISCUSSION TITLE
Where to store Frame.png

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#200] [Q&A] [answered]
First of all, I use your sample .png file

If the file in the installation folder ; it works
If I rename a in the installation folder ; it work

I tried the follwoing folders

/home/pi/photobooth-data/userdata ; no frame shown on the picture
/home/pi/photobooth-data/userdata/frames ; no frame shown on the picture

You to an other user that 
/home/pi/photobooth-data/userdata/
should work

and to me that I also can use
/home/pi/photobooth-data/userdata/frames

I tries also some variations in the config section for the folder

Did i get you wrong ?

**Comment by mgineer85** [comment, marked as answer]
Not really sure what you mean. 
upload a file just anywhere in subfolders. Example to userdata/frame you upload niceframe.png

to use it you would put userdata/frame/niceframe.png in the textfield where to use it (in the mediaprocessing sections)

helps this?

**Reply by Striker62** [reply]
Ok that was the missing part, start at /userdata.

it works . Thanks

**Reply by mgineer85** [reply]
Great! One day we will have a filepicket in the config section probably 👌

**Reply by Striker62** [reply]
now it is clear how it works. it is ok for me
